File: app/src/main/java/com/example/adondeapp/Switch.java
package com.example.adondeapp;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

public class Switch {

    private final FragmentManager fragmentManager;

    public Switch(FragmentManager fragmentManager) {
        this.fragmentManager = fragmentManager;
    }

    public void changeFragment(Fragment fragment) {
        FragmentTransaction ft = fragmentManager.beginTransaction();
        ft.replace(R.id.fragment_content, fragment);
        ft.addToBackStack(null);
        ft.commit();
    }
}

File: app/src/main/java/com/example/adondeapp/Switch.kt
package com.example.adondeapp

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager

class Switch(private val fragmentManager: FragmentManager) {
    fun changeFragment(fragment: Fragment?) {
        val ft = fragmentManager.beginTransaction()
        ft.replace(R.id.fragment_content, fragment!!)
        ft.addToBackStack(null)
        ft.commit()
    }
}
